src/renderer/components/atoms/button/button.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/button-has-type */
+import type { ButtonProps } from './types';
+import { buildStylesFor } from './helper';
+
+const Button = ({
+  className,
+  label,
+  type = 'primary',
+  variant = 'big',
+  disabled = false,
+  children,
+}: ButtonProps) => {
+  const styles = buildStylesFor(type, variant, disabled);
+
+  if (disabled) {
+    return (
+      <button className={`${styles} ${className || ''}`} disabled>
+        {label || children}
```

Comment:
let's just receive `children` and remove the `label` prop. `children` works well for both text or any other markup you need to add to the button

Reply:
done!